packages/flipper-plugin-react-navigation/src/Logs.tsx

```diff
@@ -72,14 +72,13 @@ const Row = styled.button<{ selected: boolean; faded: boolean }>((props) => ({
   'fontSize': theme.monospace.fontSize,
   'textAlign': 'left',
   'padding': `${theme.space.medium}px ${theme.space.large}px`,
-  'color': props.selected ? '#fff' : '#000',
+  'color': props.selected ? '#fff' : '',
```

Comment:
I think empty string is weird, it should specify proper color or a keyword (such as `inherit`) to make it clear what it's doing.

However, we need to migrate the plugin to the new architecture to have proper dark theme support if I understand correctly: https://fbflipper.com/docs/extending/sandy-migration/